Doc/library/stdtypes.rst

```diff
@@ -1791,8 +1792,8 @@ expression support in the :mod:`re` module).
    Return ``True`` if the string is a valid identifier according to the language
    definition, section :ref:`identifiers`.
 
-   Call :func:`keyword.iskeyword` to test whether string ``s`` is a reserved
-   identifier, such as :keyword:`def` and :keyword:`class`.
+   You can call :func:`keyword.iskeyword` to test whether string ``s`` is a
+   reserved identifier, such as :keyword:`def` and :keyword:`class`.
```

Comment:
-1 on this change. This makes it more verbose but doesn't significantly improve clarity, in my opinion. Instead, why not simply italicise the word "reserved"?

Reply:
The change made here was `Call` --> `You can call`. 
The current text can to let the reader to understand which is the `isidentifier` internally call the `iskeyword` method (mainly is is not a native speaker, I guess). But it is not true. So, the propose is to make explicit that the programmer must to call the function.

In the `reserved` was just a break line. No changed the content.

Reply:
I agree that imperative seems fine, but can you share the experiences where you found that imperative caused problems? I'd be interested in hearing the examples and seeing if they'd apply here.

Reply:
As a technical docs writer and copyeditor, imperative is generally fairly idiomatic in reference-type documentation such as this, and initially I didn't see a problem with it either.

However, looking more carefully at the context, I do see an issue with it: the imperative mood is conventionally used, as it is on the line right before, to describe what the _method_ does. However, this next sentence instead describes what _the user might want to consider doing_, and the chosen phrasing, unlike the standard imperative, makes explicit that distinction that might otherwise cause confusion, especially among readers with less native English proficiency.

Of course, such asides with suggestions for the reader aren't always appropriate in reference-type documentation, especially using language like "You can", but with a tiny bit of linguistic massaging it can fit right in (unfortunately, it won't let me actually suggest here):

```rst
   Use :func:`keyword.iskeyword` to test whether string ``s`` is a
   reserved identifier, such as :keyword:`def` and :keyword:`class`.
```

> I have experiences where the imperative was not fine. :)

I would also appreciate clarification here

Reply:
> As a technical docs writer and copyeditor, imperative is generally fairly idiomatic in reference-type documentation such as this, and initially I didn't see a problem with it either.
> 
> However, looking more carefully at the context, I do see an issue with it: the imperative mood is conventionally used, as it is on the line right before, to describe what the _method_ does. However, this next sentence instead describes what _the user might want to consider doing_, and the chosen phrasing, unlike the standard imperative, makes explicit that distinction that might otherwise cause confusion, especially among readers with less native English proficiency.

Exactly this. Especially if the native language be Portuguese.

> 
> Of course, such asides with suggestions for the reader aren't always appropriate in reference-type documentation, especially using language like "You can", but with a tiny bit of linguistic massaging it can fit right in (unfortunately, it won't let me actually suggest here):
> 
> ```
>    :func:`keyword.iskeyword` can be used to test whether string ``s`` is a
>    reserved identifier, such as :keyword:`def` and :keyword:`class`.
> ```
> 
> > I have experiences where the imperative was not fine. :)
> 
> I would also appreciate clarification here

One experience mine was: a "readers with less native English proficiency" read the actual text and understood that the method itself call the other method, not that the developer must do the call. So, is this reason of my suggestion.

Reply:
See suggestion implementing the above